app/presenters/base_presenter.rb

```diff
@@ -0,0 +1,11 @@
+class BasePresenter < SimpleDelegator
+  def initialize(model, view)
+    @view = view
+    @model = model
+    super(model)
+  end
+
+  def h
```

Comment:
So I know the blog post I linked to called this 'h', but I think a better name for this woudl be `view_context`. this could also be a simple attr_reader, and then there is no reason to write a test for it ;)